controllers/catalogController.js

```diff
@@ -0,0 +1,49 @@
+const { STATUS_CODES, ERRORS } = require('../common/constants');
+
+const trainingList = [
+    { name: 'Ноги', key: 'legs' },
+    { name: 'Руки', key: 'hands' },
+    { name: 'Силовая', key: 'strength' },
+    { name: 'Спина', key: 'back' },
+    { name: 'Грудь', key: 'chest' },
+];
+
+const tariffs = [
+    {
+        _id: "65df21ca9013cb64beacbd56",
+        name: "Pro",
+        periods: [
+            { text: "6 месяцев", cost: 5.5, days: 182 },
+            { text: "9 месяцев", cost: 8.5, days: 274 },
+            { text: "12 месяцев", cost: 10, days: 365 }
+        ]
+    }
+];

```

Comment:
you can find trainingList in ./helpers/catalogHelpers.js and use it here. Also better move tariffs in this file and use here